components/map.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Mapsearch from './mapsearch.jsx';
+
+class Map extends React.Component {
+  constructor(props) {
+    super(props);
+    this.onSearch = this.onSearch.bind(this);
+  }
+
+  componentDidMount() {
+    const theMap = this.refs.map;
+    const mapOptions = {
+      center: {lat: 37.773972,
+      lng: -122.431297},
+      zoom: 12,
+      minZoom: 3
+    };
+
+    this.initializeMap();
+  }
+
+  initializeMap() {
+    var chicago = {lat: 41.850033, lng: -87.6500523};
+
+    var INITIAL_LOCATION = {
```

Comment:
is this being used for anything. I see location data at chicago and that is being used, but this does not appear to be.